### ISSUE TITLE
Item text doesn't support an accessible alternative

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-mcq#209]
### Subject of the issue
The item `text` property sets both the option text displayed on screen as well as the `aria-label`. In the instance you want to provide an alternative text, for example, to specify how a word should be pronounced. IT to read as 'eye tea' not 'it'.

Most text properties in Adapt have an alternative property to override the property setting the on screen text.

### Expected behaviour
Provide an optional `altText` property to set the item `aria-label`. If no `altText` is set, then `text` will set the `aria-label` as per the current functionality.

Same issue raised for GMCQ - https://github.com/adaptlearning/adapt-contrib-gmcq/issues/172
